ci/test/00_setup_env_win64.sh

```diff
@@ -7,7 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_win64
-export DOCKER_NAME_TAG=ubuntu:20.04  # Check that Focal can cross-compile to win64 (Focal is used in the gitian build as well)
+export DOCKER_NAME_TAG=ubuntu:20.04  # Check that Focal can cross-compile to win64
```

Comment:
Might be good to replace with a comment to explain why Focal was picked:

LTS that offers a compiler closest to the guix-win64 compiler

Reply:
I  can address this along with another change